Initial update of elasticsearch chart upgrade (#245)
kanisterio/kanister: examples/stable/elasticsearch/README.md

```bash
# Replace the default s3 credentials (endpoint, bucket and region) with your credentials before you run this command
$ helm install stable/elasticsearch -n my-release --namespace es-test 
```

The command deploys Elasticsearch on the Kubernetes cluster in the default
configuration. 

```bash
kanctl --namespace kasten-io create profile --bucket infracloud.kanister.io --region ap-south-1 s3compliant --access-key "AKIAIOSFODNN7EXAMPLE" --secret-key "wJalrXUtnFEMI%K7MDENG%bPxRfiCYEXAMPLEKEY"
```

> This command creates a profile which we'll later use.

Reviewer: ```suggestion
This command creates a profile which we will use later.
```

This CR can be shared between Kanister-enabled application instances so one option is to

> only create as part of the first instance.

Reviewer: Lets change this to the following para since we are no longer installing profile as a part of application install:
```
The command will configure a location where artifacts resulting from Kanister
data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
can be shared between Kanister-enabled application instances.
```

Author: Updated the para as above.

> If not creating a Profile CR, the the above kanctl command can be skipped and only helm install is sufficient.

Reviewer: ```suggestion
```

Author: Removed this line.

green  open   customer xbwj34pTSZOdDI7xVR0qIA   5   1          1            0      8.9kb          4.4kb
```

## Create the Blueprint 

In order to perform backup, restore, and delete operations on the running elasticsearch, we need to create a blueprint. You can create that using the command below from the root of kanister repo.

```bash

> kubectl create -f ./examples/helm/kanister/kanister-elasticsearch/kanister/elasticsearch-blueprint.yaml -n kasten-io

Reviewer: ```suggestion
kubectl create -f ./examples/stable/elasticsearch/elasticsearch-blueprint.yaml -n kasten-io
```

Author: Made the changes to reflect correct path.
